tutorials/variational-quantum-eigensolver/vqe.ipynb

```diff
@@ -70,7 +70,7 @@
   },
```

Comment:
<p>Earlier text says:</p><blockquote>If the procedure terminates correctly, then the <code>prev_vector</code> and <code>iters</code> values in our <code>cost_history_dict</code> dictionary should be equal to the solution vector and total number of function evaluations, respectively.</blockquote><p> I.e. this should always have been <code>True</code>, not sure what happened last time this ran. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>